containers/performance_tests/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+IF(Kokkos_ENABLE_CUDA AND WIN32)
+add_compile_options(-x cu)
```

Comment:
When you read about modern CMake, the first thing they tell you is to quit using this and add target-specific options instead.

Reply:
what we need is to add it as a public compile only option to the core library somehow.

Reply:
I'm a bit confused by this. Who doesn't need this flag when building?